### ISSUE TITLE
define-component

### ISSUE THREAD

**Issue by LiberalArtist** [racket/web-server#33]
I just noticed the macro `define-component` in `web-server/web-server-lib/web-server/lang/web-cell-component.rkt`, which seems like it could be useful but doesn't seem to be documented anywhere (and it is less than immediately obvious how to use it).

Should it be documented?

**Comment by jeapostrophe** [comment]
You are right that it is not documented anywhere. It shows up in the examples:

https://github.com/racket/web-server/blob/master/web-server-lib/web-server/default-web-root/htdocs/lang-servlets/wc-comp.rkt

and I think I referenced it in a paper. Looking at it now, many many years later, it seems trivial to me and not worth being a library. If you feel differently, please feel free to document it.
